cpp/dolfinx/fem/assemble_matrix_impl.h

```diff
@@ -142,14 +142,10 @@ void assemble_matrix(
     // FIXME: cleanup these calls? Some of the happen internally again.
     mesh->topology_mutable().create_entities(tdim - 1);
```

Comment:
Could you remove create_entities from here? In `create_connectivity(tdim-1, tdim)` the connectivity is created, ref:
https://github.com/FEniCS/dolfinx/blob/c6ba579c0c09e7ec8c07d55c8c82696d55a8c961/cpp/dolfinx/mesh/Topology.cpp#L196-L197
Would address the fixme